scripts/get-rpc_release.py

```diff
@@ -22,7 +22,7 @@
 # Sourced from: https://stackoverflow.com/a/10551190
 class EnvDefault(argparse.Action):
     def __init__(self, envvar, required=True, default=None, **kwargs):
-        if not default and envvar:
+        if envvar:
```

Comment:
i suggest just dropping this check because `envvar` should always be set to something [1] significant [2].

this would make the order of preference:
1. user-provided argument
1. user-provided environment variable
1. dev-provided `default`

[1] it doesn't have a default arg value
[2] otherwise why are we using the EnvDefault class for the argparse argument action